packages/asset/contracts/AssetV2.sol

```diff
@@ -0,0 +1,31 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.18;
+
+import {Asset} from "./Asset.sol";
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import {ContextUpgradeable} from "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";
+
+/// @title AssetV2
+/// @author The Sandbox
+/// @notice ERC1155 asset token contract
+/// @notice Minting and burning tokens is only allowed through separate authorized contracts
+/// @dev This contract is final and should not be inherited
+contract AssetV2 is Asset, OwnableUpgradeable {
```

Comment:
I don't think we should add the version here, this is what i meant by using git version to handle that

Reply:
Would you put the changes in the original contract as Holly suggested above? Otherwise the new contract has to have a different name.